src/main/docker/Dockerfile

```diff
@@ -63,7 +67,21 @@ COPY --from=os1-image --chown=1000:0 /usr/share/opensearch/ /usr/share/opensearc
 
 # Configure logging
 COPY log4j2.properties /usr/share/opensearch/config
-COPY /maven/*.jar /usr/share/opensearch/lib/
+COPY /maven/log4j-*.jar /usr/share/opensearch/lib/
```

Comment:
We're missing a few jars.

Reply:
Fixed here: https://github.com/CAFapi/opensuse-opensearch1-image/pull/38/commits/1f90fb2b9c721e9099b6975b56e066db35b13d38